src/assets/config/config.deploy.json

```diff
@@ -27,6 +27,7 @@
     "APITechnicalRecordServerUri": "https://#{apiServerPrefix}.demo.com/api/v1.0/",
     "APITestResultServerUri": "https://#{apiServerPrefix}.demo.com/api/v1.0/",
     "APIDocumentsServerUri": "https://api.nonprod.cvs.dvsacloud.uk/BRANCH",
```

Comment:
remove from commit and rewrite history please.

Reply:
I think this one should stay as I added APIPreparersServerUri and APITestStationsServerUri for deploy. We are using Preparers and Test Stations services to populate the autocomplete fields.

Reply:
Did you check with Tanio? I believe we have the API urls configured dynamically in the pipeline as environments variables.

Reply:
Yes I checked with Temi, he said to add them like this.

Reply:
thanks, resolving if is the case.